### PR TITLE
feat(oui-stepper): emit event on form submit

### PR DESCRIPTION
**Pull request by marie-j**
### Benefits

Allow to associate global action on step form submission

### CODE REVIEW

File: packages/oui-stepper/src/step-form/step-form.controller.js [hunk 1]
```diff
@@ -64,6 +64,7 @@ export default class StepFormController {
     }
 
     onFormSubmit (form) {
+        this.$scope.$emit("oui-step-form.submit", { form });
```
Comment (antleblanc) [inline packages/oui-stepper/src/step-form/step-form.controller.js]: Needed even if form is `$invalid`?

**Comment by AxelPeter** [comment]
Why this need to be in the UI Kit ?

This can already be done with `on-submit="$ctrl.onSubmit(form)"`.
And $emit in your project with your callback. ;)

**Comment by marie-j** [comment]
@AxelPeter I want to apply the same action (here tracking) on each steps submission. 
If I use the `onSubmit` callback I have to add it for each step form manually and also will have to add it for further added step-forms. By adding an event and listening to it in the hosts (if needed of course) we don't have add it manually each time

File: packages/oui-stepper/src/step-form/step-form.controller.js (hunk 1 above)
Comment (marie-j) [reply packages/oui-stepper/src/step-form/step-form.controller.js]: Yes it is

**Pull request merged**